app/models/invenio_rdm_record_converter.rb

```diff
@@ -375,6 +375,11 @@ def format_publication_date(publication_date)
 
   def normalize_date(date_string)
     split_date = date_string.split(/[-,\/ ]/).map(&:downcase)
+    # date format starts with month first
+    if (!split_date.blank? && split_date[0].length < 3)
```

Comment:
Thanks for figuring this out! I was having a really hard time with these kinds of dates previously.

Reply:
Thanks to you for getting this started! I only added a small piece, while you did the heavy lifting with the dates.